.vscode/launch.json

```
@@ -35,6 +35,17 @@
 					"path": "${workspaceFolder}/client"
 				}
 			]
+		},
+		{
+			"name": "Debug Current Jest File",
+			"type": "node",
+			"request": "launch",
+			"runtimeExecutable": "npm",
+			"runtimeArgs": [ "run", "test-client" ],
```

Since we use Yarn, might be better to swap this with equivalent `yarn` command

```suggestion
			"runtimeExecutable": "yarn",
			"runtimeArgs": [ "test-client" ],
```